tests/sqs_service.py

```diff
@@ -1,15 +1,16 @@
 import requests
 
-SQS_STAGE = "https://sqs.stage.osmosis.zone"
+SQS_STAGE = "http://localhost:9092"
```

Comment:
Note to reviewer: I will change this back before merge

Reply:
🆗